groovy/libs.groovy/src/org/netbeans/modules/libs/groovy/groovy.xml

```diff
@@ -28,6 +28,7 @@
     <volume>
         <type>classpath</type>
         <resource>jar:nbinst://org.netbeans.modules.libs.groovy/modules/ext/groovy.jar!/</resource>
+        <resource>jar:nbinst://org.netbeans.modules.libs.groovy/modules/ext/groovy-ant.jar!/</resource>
```

Comment:
Please also update the properties to:
```xml
        <property>
             <name>maven-dependencies</name>
             <value>
                 org.codehaus.groovy:groovy:2.5.2:jar
                 org.codehaus.groovy:groovy-ant:2.5.2:jar
             </value>
          </property>
```
Then the properties match the classpath

Reply:
Will do.